controllers/bootstrap/init.go

```diff
@@ -180,6 +180,15 @@ func (b *Bootstrap) InitResources(instance *apiv3.CommonService) error {
 		}
 	}
 
+	// Check storageClass
+	csStorageClass, err := util.GetStorageClass(b.Reader)
+	if err == nil {
```

Comment:
I think we should handle the situation that the error happens when we get storage class, for example
```
if err != nil {
     return fmt.Errorf("Failed to get StorageClass")
}
```
and after the error checking, we could validate the storage class